Tasks/DeployVisualStudioTestAgent/TestAgentHelper.ps1

```diff
@@ -204,5 +204,24 @@ namespace CapabilityHelpers.VisualStudio.Setup.Com
     }
 }
 
-
-
+function Remove-Service([String] $ServiceName)
+{
+    if(Get-Service $ServiceName -ErrorAction SilentlyContinue)
+    {
+        $service = (Get-WmiObject Win32_Service -filter "name='$ServiceName'")
+        Write-Verbose -Message("Trying to delete service {0}" -f $ServiceName) -Verbose
+        if($service)
+        {
+            $service.StopService()
```

Comment:
> $service.StopService() [](start = 11, length = 23)

Please check, what happens, if stopping the service throws an error, as we are not handling that?